src/internal/i18n/context.ts

```diff
@@ -11,11 +11,9 @@ export interface FormatFunction {
   <T>(namespace: string, component: string, key: string, provided: T, handler?: CustomHandler<T>): T;
 }
 
-function defaultFormatFunction<T>(_namespace: string, _component: string, _key: string, provided: T) {
-  return provided;
-}
-
-export const InternalI18nContext = React.createContext<FormatFunction>(defaultFormatFunction);
+export const InternalI18nContext = React.createContext<FormatFunction>(
+  <T>(_namespace: string, _component: string, _key: string, provided: T) => provided
```

Comment:
Does it need explicit argument types when the `createContext<FormatFunction>` is already typed

Reply:
:(